styleguide/config.js

```diff
@@ -5,6 +5,16 @@ const { reactDocgenTypescript } = require("./propsParser.config");
 const baseConfig = {
   title: "VKUI styleguide",
   styleguideDir: path.join(__dirname, `../${argv.dist || "styleguide-build"}`),
+  template: {
+    head: {
+      links: [
```

Comment:
гугл еще рекомендует запихнуть

```html
<link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
```

```suggestion
      links: [
        {
          rel: "preconnect",
          href: "https://fonts.googleapis.com",
        },
        {
          rel: "preconnect",
          href: "https://fonts.gstatic.com",
          crossorigin: "anonymous",
        },
```